docs/tutorials/tutorial_experience.ipynb

```diff
@@ -0,0 +1,859 @@
+{
```

Comment:
<p>I think this a really interesting point to include in the tutorial, very nice! I would rephrase this a bit though to make it more clear that we are actively decreasing the return to fishing here to consider a different parameter specification because the return to experience for fishing is what's driving the above effect of initial experience. Otherwise one might wonder where the new example is coming from, but maybe that's just the way I read it :).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenSourceEconomics/respy/pull/375/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Agree